src/go/configinfo/service_info.go

```diff
@@ -243,27 +242,30 @@ func (s *ServiceInfo) addGrpcHttpRules() {
 
 func (s *ServiceInfo) processAccessToken() {
 	if s.Options.ServiceAccountKey != "" {
-		data, _ := ioutil.ReadFile(s.Options.ServiceAccountKey)
 		s.AccessToken = &commonpb.AccessToken{
-			TokenType: &commonpb.AccessToken_ServiceAccountSecret{
-				ServiceAccountSecret: &commonpb.DataSource{
-					Specifier: &commonpb.DataSource_InlineString{
-						InlineString: string(data),
-					},
+			TokenType: &commonpb.AccessToken_RemoteToken{
+				RemoteToken: &commonpb.HttpUri{
+					// Use the same path as IMDS to get access token.
+					Uri:     fmt.Sprintf("http://%s:%v%s", util.LoopbackIPv4Addr, s.Options.LocalAccessTokenServerPort, util.AccessTokenPath),
```

Comment:
Why are we using the same access token path as metadata server?   I could not come up any reason why not.  But still prefer not to use the same path.    how about just /local/access_token?

Reply:
Curious about this too. I don't see any need for using the same path, we just need to maintain the same response format. Might make debugging easier too if we clearly distinguish between the paths.